packages/components/src/TestComponent/TestComponent.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactNode } from "react";
```

Comment:
This is a file generated from the script for the purpose of reviewing. TODO: remove this file before merging this PR.